src/engine/ExportQueryExecutionTrees.cpp

```diff
@@ -180,9 +180,9 @@ ExportQueryExecutionTrees::constructQueryResultToTriples(
        getRowIndices(limitAndOffset, *result, resultSize)) {
     auto& idTable = pair.idTable_;
     for (uint64_t i : range) {
-      ConstructQueryExportContext context{i, idTable, pair.localVocab_,
-                                          qet.getVariableColumns(),
-                                          qet.getQec()->getIndex()};
+      ConstructQueryExportContext context{
+          resultSize + i - range.front(), idTable, pair.localVocab_,
+          qet.getVariableColumns(), qet.getQec()->getIndex()};
```

Comment:
What does this change in the first argument here do, and is this the position that breaks the conformance tests?